modules/administration-guide/partials/proc_viewing-dev-workspace-operator-metrics-on-grafana-dashboards.adoc

```diff
@@ -1,31 +1,30 @@
 [id="proc_viewing-dev-workspace-operator-metrics-on-grafana-dashboards"]
-= Viewing {devworkspace} Operator metrics on Grafana dashboards
+= Viewing {devworkspace} Operator metrics from an OpenShift web console dashboard
 
-To view the {devworkspace} Operator metrics on Grafana with the example dashboard:
+After configuring the in-cluster Prometheus instance to collect {devworkspace} Operator metrics, you can view the metrics on a custom dashboard in the *Administrator* perspective of the OpenShift web console.
 
 .Prerequisites
 
-* Prometheus is collecting metrics. See xref:proc_collecting-dev-workspace-operator-metrics-with-prometheus[].
-
-* Grafana version 7.5.3 or later.
-
-* Grafana is running on port `3000` with a corresponding Service. See link:https://grafana.com/docs/grafana/latest/installation/kubernetes/[Installing Grafana].
+* An active `{orch-cli}` session with administrative permissions to the destination {orch-name} cluster. See {orch-cli-link}.
 
+* An instance of {prod-short} running in {orch-name}.
 
+* The in-cluster Prometheus instance is collecting metrics. See xref:proc_collecting-dev-workspace-operator-metrics-with-prometheus[].
 
 .Procedure
 
-. Add the data source for the Prometheus instance.
-See link:https://prometheus.io/docs/visualization/grafana/#creating-a-prometheus-data-source[Creating a Prometheus data source].
-
-. Import the link:https://github.com/devfile/devworkspace-operator/blob/main/docs/grafana/grafana-dashboard.json[example `grafana-dashboard.json`] dashboard.
-
+. Create a ConfigMap in the `openshift-config-managed` {orch-namespace} containing the dashboard definition and apply the necessary label. The dashboard definition is based on Grafana 6.x dashboards, however, not all Grafana 6.x dashboard features are supported within the OpenShift web console.
++
+[source,terminal,subs="+attributes,quotes"]
+----
+$ oc create configmap grafana-dashboard-dwo \
+  --from-literal=dwo-dashboard.json="$(curl https://raw.githubusercontent.com/devfile/devworkspace-operator/main/docs/grafana/openshift-console-dashboard.json)" \
+  -n openshift-config-managed
+$ oc label configmap grafana-dashboard-dwo console.openshift.io/dashboard=true -n openshift-config-managed
+----
 
 .Verification steps
 
-* Use the Grafana console to view the {devworkspace} Operator metrics dashboard. See xref:ref_grafana-dashboards-for-the-dev-workspace-operator[].
-
-.Additional resources
+. In the *Administrator* view of the OpenShift web console, navigate to *Observe* -> *Dashboards* from the sidebar menu on the left.
```

Comment:
Our style guide recommends avoiding positional references in the UI like "the sidebar menu on the left", also IBM Style advises avoiding "navigate":
```suggestion
. In the *Administrator* view of the OpenShift web console, go to *Observe* -> *Dashboards*.
```